src/contracts/polygon/AaveV3PolCapsPayload-Feb12.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {AaveV3Polygon} from 'aave-address-book/AaveV3Polygon.sol';
+import {IPoolConfigurator, ConfiguratorInputTypes} from 'aave-address-book/AaveV3.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @dev This payload sets borrow caps for multiple assets on AAVE V3 Polygon
+ * - Dicussion: https://governance.aave.com/t/arc-chaos-labs-supply-and-borrow-cap-updates-aave-v3-polygon-and-arbitrum-2023-02-07/11605
+ */
+contract AaveV3PolCapsPayload is IProposalGenericExecutor {
+  address public constant BAL = 0x9a71012B13CA4d3D0Cdc72A177DF3ef03b0E76A3;
```

Comment:
you can fetch asset addresses from address book to prevent copy paste errors etc https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV3Polygon.sol#L61

Reply:
Oh, nice! that's helpful